### PR TITLE
Feature/source data as turtle

### PR DESCRIPTION
**Pull request by kwahlin**
`spraakkoder.tsv` and `landskoder.tsv` translated to Turtle. Should we do the same with `nationalitetskoder.tsv`?

Resources with obsolete codes have been removed. The obsolete code is instead included in the "new" resource, which also points to the removed resource with `sameAs`. Problem: how to know which code is the current?

### CODE REVIEW

**Comment by olovy** [comment]
* Östsamiska should actually be `replacedBy` sms, smn, smi
* mol should actually be `replacedBy` ron
* Meänkieli does not have an ISO 639-2 code and should use Libris-defined 9mk in MARC
* Fornlågtyska does not have an ISO 639-2 code and should use Libris-defined 9ft in MARC

See fa16663

**Comment by olovy** [comment]
> Problem: how to know which code is the current?

If we use `langCode` for conversion to MARC this should be fine?
When converting from MARC, any code in `code` should be picked up by the normalization step so we will be able to handle any obsolete Libris-defined codes.

File: source/languages.ttl [hunk 1]
```diff
@@ -0,0 +1,3174 @@
...
+<que> a schema:Language ;
+    skos:notation "que",
+        "que"^^kbv:ISO639-2,
+        "que"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Quechua"@sv .
+
+<raj> a schema:Language ;
+    owl:sameAs <rai> ;
+    skos:notation "rai",
+        "raj",
```
Comment (olovy) [inline source/languages.ttl]:
```suggestion
    skos:notation "raj",
```

From https://github.com/libris/definitions/blob/develop/source/spraakkoder.tsv#L366

I can't find any reference to this code in 
https://www.loc.gov/standards/iso639-2/php/code_changes.php
https://www.loc.gov/marc/languages/language_code.html
https://www.loc.gov/marc/languages/languagechg.html
https://katalogverk.kb.se/katalogisering/Formathandboken/Sprakkoder/index.html

Is it an ancient MARC code? I suggest we just drop it.

**Comment by kwahlin** [comment]
> > Problem: how to know which code is the current?
> 
> If we use `langCode` for conversion to MARC this should be fine? When converting from MARC, any code in `code` should be picked up by the normalization step so we will be able to handle any obsolete Libris-defined codes which are defined there, e.g. 9ss.

Right, no problem then!

File: source/countries.ttl [hunk 2]
```diff
@@ -0,0 +1,1927 @@
...
+@prefix schema: <http://schema.org/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix wd: <http://www.wikidata.org/entity/> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+@base <https://id.kb.se/country/> .
+
+<9a> a schema:Country ;
+    rdfs:comment "Librisdefinierad kod"@sv ;
```
Comment (olovy) [inline source/countries.ttl:15]:
```suggestion
    rdfs:comment "Librisdefinierad kod"@sv ;
    skos:exactMatch wd:Q5689 ;
```

Comment (kwahlin) [reply source/countries.ttl:15]: Done in 9b8e1ee

**Comment by kwahlin** [comment]
f3989c6 requires some cleaning up, see https://jira.kb.se/browse/LXL-3778

**Comment by kwahlin** [comment]
Constructing nationalities is very straight-forward, can't no reason to not do it. See 67490c5

**Comment by kwahlin** [comment]
I couldn't resist converting the last few tsv's as well :grimacing:  As for the RDA terms (9483e1d) I don't think there is too much to say, however the conversion from `funktionskoder.tsv` to `relators.ttl` comes with some changes:
- `:closeMatch` to RDA terms have been included as per https://jira.kb.se/browse/LXL-3768
- _All_ labels in `funktionskoder.tsv` except Sweidsh have been moved to `:hiddenLabel` (so that they can still be used for normalization), instead we construct the "real" labels from respective external source.
- English labels from http://id.loc.gov/vocabulary/relators
- Finnish from [MTS](http://finto.fi/mts/sv/page/m3414) (we should try and map more of these)
- German from [GND](https://d-nb.info/standards/elementset/gnd) (we should try and map more of these)
- French from [BNF](https://data.bnf.fr/vocabulary/roles) (no mappings yet)
- The French relator terms (all together) are only available as RDF via https://data.bnf.fr/sparql/ as far as can tell, hence fb43bef. This is dependant on https://github.com/libris/definitions/pull/363 to work..
- We can probably clean up some superfluous hiddenLabels when we see what labels we get from the external sources.

Feel free to shoot this down :smiley:

**Comment by niklasl** [comment]
This seems great! Are there any remaining outstanding questions regarding codes?

Also, we should carefully verify that the output is identical (apart from added or intentional changes of course), unless you've already done so?

One method to check output changes is:
- stash any ongoing stuff and checkout develop
- run `PYTHONHASHSEED=1 python datasets.py`
- move the resulting build directory aside, `mv build /tmp/defs-build-1`
- check out the branch to test (i.e. this feature)
- then run `PYTHONHASHSEED=1 python datasets.py` again
- and compare using `diff -qr /tmp/defs-build-1 build`

**Comment by kwahlin** [comment]
> This seems great! Are there any remaining outstanding questions regarding codes?
> 
> Also, we should carefully verify that the output is identical (apart from added or intentional changes of course), unless you've already done so?
> 
> One method to check output changes is:
> 
>     * stash any ongoing stuff and checkout develop
> 
>     * run `PYTHONHASHSEED=1 python datasets.py`
> 
>     * move the resulting build directory aside, `mv build /tmp/defs-build-1`
> 
>     * check out the branch to test (i.e. this feature)
> 
>     * then run `PYTHONHASHSEED=1 python datasets.py` again
> 
>     * and compare using `diff -qr /tmp/defs-build-1 build`

I don't think we're almost good to go. Just need confirmation from the metadata people that it's a go on language and country code changes. Definitely need to properly verify the output too, haven't done that yet. Thanks for the tip.

File: source/languages.ttl [hunk 3]
```diff
@@ -2530,8 +2530,7 @@
 
 <sju> a schema:Language ;
     owl:sameAs <9su> ;
```
Comment (niklasl) [inline source/languages.ttl:2532]: Maybe add a `skos:historyNote` (or just an `rdfs:comment`) to make it clear that once it time, `"9su"` was used as a code for this? (An `owl:sameAs` is "opaque" and won't be accessible in UI views.)

**Comment by niklasl** [comment]
We need something like:
```json
    "langCodeMarc": {"@id": "code", "@type": "MarcLanguageCode"},
    "langCodeLibrisMarc": {"@id": "code", "@type": "LibrisMarcLanguageCode"},
```
in `sys/context/base.jsonld` to get these type code objects out of the way of the plain-string code property.

(Otherwise the data mixes strings and objects, which is formally OK, but "uncomfortable" in general, and breaks Elastic.)

File: source/languages.ttl [hunk 4]
```diff
@@ -0,0 +1,3171 @@
...
+
+<gem> a schema:Language ;
+    skos:notation "gem",
+        "gem"^^kbv:ISO639-2,
+        "gem"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Germanska språk (övriga)"@sv .
+
+<geo> a schema:Language ;
+    skos:notation "geo",
+        "geo"^^kbv:ISO639-2,
```
Comment (olovy) [inline source/languages.ttl:989]:
```suggestion
        "geo"^^kbv:ISO639-2,
        "kat"^^kbv:ISO639-2,
```

Comment (olovy) [reply source/languages.ttl:989]: ISO-639-2 = ISO-639-2/B ∪  ISO-639-2/T so both codes should always exist as `^^kbv:ISO639-2`

File: source/languages.ttl [hunk 5]
```diff
@@ -0,0 +1,3171 @@
...
+    rdfs:comment "-1453"@sv ;
+    skos:notation "grc",
+        "grc"^^kbv:ISO639-2,
+        "grc"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Grekiska, klassisk"@sv .
+
+<gre> a schema:Language ;
+    rdfs:comment "1453-"@sv ;
+    skos:notation "gre",
+        "gre"^^kbv:ISO639-2,
```
Comment (olovy) [inline source/languages.ttl:1091]:
```suggestion
        "gre"^^kbv:ISO639-2,
        "ell"^^kbv:ISO639-2,
```

File: source/languages.ttl [hunk 6]
```diff
@@ -0,0 +1,3171 @@
...
+<ibo> a schema:Language ;
+    skos:altLabel "Igbo"@sv ;
+    skos:notation "ibo",
+        "ibo"^^kbv:ISO639-2,
+        "ibo"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Ibo"@sv .
+
+<ice> a schema:Language ;
+    skos:notation "ice",
+        "ice"^^kbv:ISO639-2,
```
Comment (olovy) [inline source/languages.ttl:1233]:
```suggestion
        "ice"^^kbv:ISO639-2,
        "isl"^^kbv:ISO639-2,
```

File: source/languages.ttl [hunk 7]
```diff
@@ -0,0 +1,3171 @@
...
+
+<man> a schema:Language ;
+    skos:notation "man",
+        "man"^^kbv:ISO639-2,
+        "man"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Mande"@sv .
+
+<mao> a schema:Language ;
+    skos:notation "mao",
+        "mao"^^kbv:ISO639-2,
```
Comment (olovy) [inline source/languages.ttl:1766]:
```suggestion
        "mao"^^kbv:ISO639-2,
        "mri"^^kbv:ISO639-2,
```

File: source/languages.ttl [hunk 8]
```diff
@@ -0,0 +1,3171 @@
...
+<mas> a schema:Language ;
+    skos:altLabel "Massaj"@sv ;
+    skos:notation "mas",
+        "mas"^^kbv:ISO639-2,
+        "mas"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Maa"@sv .
+
+<may> a schema:Language ;
+    skos:notation "may",
+        "may"^^kbv:ISO639-2,
```
Comment (olovy) [inline source/languages.ttl:1793]:
```suggestion
        "may"^^kbv:ISO639-2,
        "msa"^^kbv:ISO639-2,
```

File: source/languages.ttl [hunk 9]
```diff
@@ -0,0 +1,3171 @@
...
+<peo> a schema:Language ;
+    rdfs:comment "Ca 600-400 f.Kr."@sv ;
+    skos:notation "peo",
+        "peo"^^kbv:ISO639-2,
+        "peo"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Fornpersiska"@sv .
+
+<per> a schema:Language ;
+    skos:notation "per",
+        "per"^^kbv:ISO639-2,
```
Comment (olovy) [inline source/languages.ttl:2243]:
```suggestion
        "per"^^kbv:ISO639-2,
        "fas"^^kbv:ISO639-2,
```

File: source/languages.ttl [hunk 10]
```diff
@@ -0,0 +1,3171 @@
...
+
+<rom> a schema:Language ;
+    skos:notation "rom",
+        "rom"^^kbv:ISO639-2,
+        "rom"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Romani"@sv .
+
+<rum> a schema:Language ;
+    skos:notation "rum",
+        "rum"^^kbv:ISO639-2,
```
Comment (olovy) [inline source/languages.ttl:2353]:
```suggestion
        "rum"^^kbv:ISO639-2,
        "ron"^^kbv:ISO639-2,
```

File: source/languages.ttl [hunk 11]
```diff
@@ -0,0 +1,3171 @@
...
+
+<sla> a schema:Language ;
+    skos:notation "sla",
+        "sla"^^kbv:ISO639-2,
+        "sla"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Slaviskt språk"@sv .
+
+<slo> a schema:Language ;
+    skos:notation "slo",
+        "slo"^^kbv:ISO639-2,
```
Comment (olovy) [inline source/languages.ttl:2546]:
```suggestion
        "slo"^^kbv:ISO639-2,
        "slk"^^kbv:ISO639-2,
```

File: source/languages.ttl [hunk 12]
```diff
@@ -0,0 +1,3171 @@
...
+
+<tha> a schema:Language ;
+    skos:notation "tha",
+        "tha"^^kbv:ISO639-2,
+        "tha"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Thailändska"@sv .
+
+<tib> a schema:Language ;
+    skos:notation "tib",
+        "tib"^^kbv:ISO639-2,
```
Comment (olovy) [inline source/languages.ttl:2808]:
```suggestion
        "tib"^^kbv:ISO639-2,
        "bod"^^kbv:ISO639-2,
```

File: source/languages.ttl [hunk 13]
```diff
@@ -0,0 +1,3171 @@
...
+
+<was> a schema:Language ;
+    skos:notation "was",
+        "was"^^kbv:ISO639-2,
+        "was"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Washo"@sv .
+
+<wel> a schema:Language ;
+    skos:notation "wel",
+        "wel"^^kbv:ISO639-2,
```
Comment (olovy) [inline source/languages.ttl:3050]:
```suggestion
        "wel"^^kbv:ISO639-2,
        "cym"^^kbv:ISO639-2,
```

File: source/languages.ttl [hunk 14]
```diff
@@ -0,0 +1,3171 @@
...
+
+<akk> a schema:Language ;
+    skos:notation "akk",
+        "akk"^^kbv:ISO639-2,
+        "akk"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Akkadiska"@sv .
+
+<alb> a schema:Language ;
+    skos:notation "alb",
+        "alb"^^kbv:ISO639-2,
```
Comment (olovy) [inline source/languages.ttl:109]:
```suggestion
        "alb"^^kbv:ISO639-2,
        "sqi"^^kbv:ISO639-2,
```

File: source/languages.ttl [hunk 15]
```diff
@@ -0,0 +1,3171 @@
...
+
+<arg> a schema:Language ;
+    skos:notation "arg",
+        "arg"^^kbv:ISO639-2,
+        "arg"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Aragonska"@sv .
+
+<arm> a schema:Language ;
+    skos:notation "arm",
+        "arm"^^kbv:ISO639-2,
```
Comment (olovy) [inline source/languages.ttl:179]:
```suggestion
        "arm"^^kbv:ISO639-2,
        "hye"^^kbv:ISO639-2,
```

File: source/languages.ttl [hunk 16]
```diff
@@ -0,0 +1,3171 @@
...
+
+<ban> a schema:Language ;
+    skos:notation "ban",
+        "ban"^^kbv:ISO639-2,
+        "ban"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Balinesiska"@sv .
+
+<baq> a schema:Language ;
+    skos:notation "baq",
+        "baq"^^kbv:ISO639-2,
```
Comment (olovy) [inline source/languages.ttl:308]:
```suggestion
        "baq"^^kbv:ISO639-2,
        "eus"^^kbv:ISO639-2,
```

File: source/languages.ttl [hunk 17]
```diff
@@ -0,0 +1,3171 @@
...
+
+<bul> a schema:Language ;
+    skos:notation "bul",
+        "bul"^^kbv:ISO639-2,
+        "bul"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Bulgariska"@sv .
+
+<bur> a schema:Language ;
+    skos:notation "bur",
+        "bur"^^kbv:ISO639-2,
```
Comment (olovy) [inline source/languages.ttl:449]:
```suggestion
        "bur"^^kbv:ISO639-2,
        "mya"^^kbv:ISO639-2,
```

File: source/languages.ttl [hunk 18]
```diff
@@ -0,0 +1,3171 @@
...
+
+<chg> a schema:Language ;
+    skos:notation "chg",
+        "chg"^^kbv:ISO639-2,
+        "chg"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Chagatai"@sv .
+
+<chi> a schema:Language ;
+    skos:notation "chi",
+        "chi"^^kbv:ISO639-2,
```
Comment (olovy) [inline source/languages.ttl:531]:
```suggestion
        "chi"^^kbv:ISO639-2,
        "zho"^^kbv:ISO639-2,
```

File: source/languages.ttl [hunk 19]
```diff
@@ -0,0 +1,3171 @@
...
+
+<cus> a schema:Language ;
+    skos:notation "cus",
+        "cus"^^kbv:ISO639-2,
+        "cus"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Kusjitiska språk (övriga)"@sv .
+
+<cze> a schema:Language ;
+    skos:notation "cze",
+        "cze"^^kbv:ISO639-2,
```
Comment (olovy) [inline source/languages.ttl:676]:
```suggestion
        "cze"^^kbv:ISO639-2,
        "ces"^^kbv:ISO639-2,
```

File: source/languages.ttl [hunk 20]
```diff
@@ -0,0 +1,3171 @@
...
+<dum> a schema:Language ;
+    rdfs:comment "Ca 1050-1350"@sv ;
+    skos:notation "dum",
+        "dum"^^kbv:ISO639-2,
+        "dum"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Medelnederländska"@sv .
+
+<dut> a schema:Language ;
+    skos:notation "dut",
+        "dut"^^kbv:ISO639-2,
```
Comment (olovy) [inline source/languages.ttl:771]:
```suggestion
        "dut"^^kbv:ISO639-2,
        "nld"^^kbv:ISO639-2,
```

File: source/languages.ttl [hunk 21]
```diff
@@ -0,0 +1,3171 @@
...
+
+<fon> a schema:Language ;
+    skos:notation "fon",
+        "fon"^^kbv:ISO639-2,
+        "fon"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Fon"@sv .
+
+<fre> a schema:Language ;
+    skos:notation "fre",
+        "fre"^^kbv:ISO639-2,
```
Comment (olovy) [inline source/languages.ttl:913]:
```suggestion
        "fre"^^kbv:ISO639-2,
        "fra"^^kbv:ISO639-2,
```

File: source/languages.ttl (hunk 14 above)
Comment (olovy) [reply source/languages.ttl:109]: ISO-639-2 = ISO-639-2/B ∪ ISO-639-2/T so both B and T should exist as `^^kbv:ISO639-2`

File: source/languages.ttl (hunk 4 above)
Comment (olovy) [reply source/languages.ttl:989]: We have all three `kbv:ISO639-2`, `bv:ISO639-2-T`, `bv:ISO639-2-B`

File: source/languages.ttl (hunk 1 above)
Comment (klngwll) [reply source/languages.ttl:2315]: There is no usage in Libris so I suggest drop but keep the rai comment as a historyNote.

File: source/languages.ttl (hunk 4 above)
Comment (niklasl) [reply source/languages.ttl:989]: Let's drop the redundant `kbv:ISO639-2` if there are specific `kbv:ISO639-2-B` and/or `kbv:ISO639-2-T` codes (those types are already defined as `rdfs:subClassOf :ISO639-2`).

File: source/vocab/details.ttl [hunk 22]
```diff
@@ -619,6 +619,14 @@
     rdfs:seeAlso <https://iso639-3.sil.org/about/relationships>;
     owl:equivalentClass dc:ISO639-3 .
 
+:LibrisMarcLanguageCode a rdfs:Datatype ;
+    rdfs:label "Librisdefinierad MARC-språkkod" ;
+    rdfs:subClassOf :Local .
+
+:MarcLanguageCode a rdfs:Datatype ;
+    rdfs:label "MARC-språkkod" .
```
[the remark this answers is not in the document]
Comment (klngwll) [reply source/vocab/details.ttl:627]: I did, good catch 😁

File: source/languages.ttl [hunk 23]
```diff
@@ -0,0 +1,3171 @@
...
+    skos:prefLabel "Forndanska"@sv .
+
+<9fs> a schema:Language ;
+    rdfs:comment "-1500 (Librisdefinierad kod)"@sv ;
+    skos:notation "9fs",
+        "9fs"^^kbv:LibrisMarcLanguageCode ;
+    skos:prefLabel "Fornsvenska"@sv .
+
+<9so> a schema:Language ; 
+    rdfs:comment "Obsolet Librisdefinierad kod: använd \"smn\" (enaresamiska), \"sms\" (skoltsamiska) eller \"smi\" (samiska)"@sv ;
```
Comment (olovy) [inline source/languages.ttl:26]:
```suggestion
    rdfs:comment "Obsolet: använd \"smn\" (enaresamiska), \"sms\" (skoltsamiska) eller \"smi\" (samiska)"@sv ;
```

File: source/languages.ttl [hunk 24]
```diff
@@ -0,0 +1,3171 @@
...
+
+<fin> a schema:Language ;
+    skos:notation "fin",
+        "fin"^^kbv:ISO639-2,
+        "fin"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Finska"@sv .
+
+<fit> a schema:Language ;
+    owl:sameAs <9mk> ;
+    rdfs:comment "9mk Librisdefinierad MARC-kod"@sv ;
```
Comment (olovy) [inline source/languages.ttl:892]:
```suggestion
    rdfs:comment "(\"9mk\" Librisdefinierad MARC-kod)"@sv ;
```

File: source/languages.ttl [hunk 25]
```diff
@@ -0,0 +1,3171 @@
...
+@base <https://id.kb.se/language/> .
+
+<9fd> a schema:Language ;
+    rdfs:comment "-1500. \"9fd\" Librisdefinierad MARC-kod."@sv ;
+    skos:notation "9fd",
+        "9fd"^^kbv:LibrisMarcLanguageCode ;
+    skos:prefLabel "Forndanska"@sv .
+
+<9fs> a schema:Language ;
+    rdfs:comment "-1500.  \"9fs\" Librisdefinierad MARC-kod."@sv ;
```
Comment (olovy) [inline source/languages.ttl:20]:
```suggestion
    rdfs:comment "-1500. \"9fs\" Librisdefinierad MARC-kod."@sv ;
```

File: source/languages.ttl [hunk 26]
```diff
@@ -0,0 +1,3171 @@
...
+
+<fin> a schema:Language ;
+    skos:notation "fin",
+        "fin"^^kbv:ISO639-2,
+        "fin"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Finska"@sv .
+
+<fit> a schema:Language ;
+    owl:sameAs <9mk> ;
+    rdfs:comment "(\"9mk\" Librisdefinierad MARC-kod)"@sv ;
```
Comment (olovy) [inline source/languages.ttl:892]:
```suggestion
    rdfs:comment "\"9mk\" Librisdefinierad MARC-kod"@sv ;
```

File: source/languages.ttl [hunk 27]
```diff
@@ -0,0 +1,3171 @@
...
```
Comment (olovy) [inline source/languages.ttl]:
```suggestion
    rdfs:comment "\"9mk\" Librisdefinierad MARC-kod"@sv ;
```

File: source/vocab/details.ttl (hunk 22 above)
[the remark this answers is not in the document]
Comment (kwahlin) [reply source/vocab/details.ttl]: No need for this datatype.

**Review by niklasl — APPROVED** [review]
:shipit:

**Review by olovy — COMMENTED** [review]
About the relator labels from RDA.
These have been moved from `altLabel` to `hiddenLabel` which I think is fine.
However we've lost the connection to RDA. At least add a comment for each string in the `.ttl`? 

```
<writerOfAddedCommentary> a bf:Role ;
    rdfs:domain :Work ;
    owl:equivalentProperty locrel:wac ;
    owl:sameAs <wac> ;
    skos:exactMatch locrel:wac,
        gnd:writerOfAddedCommentary ;
    skos:hiddenLabel "writer of supplementary textual content" ;
    skos:notation "wac" ;
    skos:prefLabel "Analys, kommentar"@sv .
```

Old funktionskoder.tsv:
```
code	rda_app_i_1_en	rda_app_i_2_en	rda_app_i_3_en
abr	abridger		
act	actor		
adi	art director		
adp	adaptor		
anm	animator		
ann	annotator		
ape	appellee		
apl	appellant		
arc	architect		
arr	arranger		
art	artist		
ato	autographer		
aui	writer of introduction	writer of preface	writer of supplementary textual content
aus	screenwriter		
aut	author		
bkd	book designer		
bnd	binder		
brd	broadcaster		
brl	braille embosser		
cas	caster		
chr	choreographer	choreographer (expression)	
clt	collotyper		
cmm	commentator		
cmp	composer	composer (expression)	
cnd	conductor		
cng	cirector of photography		
col	collector		
com	compiler		
cor	collection registrar		
cou	court governed		
crt	court reporter		
csl	consultant		
cst	costume designer		
ctg	cartographer		
cur	curator		
dfd	defendant		
dgg	degree granting institution		
dnc	dancer		
drm	draftsman		
drt	director		
dsr	designer		
dte	dedicatee	dedicatee of item	
edc	editor of compilation		
edm	editor of moving image work		
edt	editor		
egr	engraver		
enj	enacting jurisdiction		
etr	etcher		
fds	film distributor		
fmd	film director		
fmk	filmmaker		
fmo	former owner		
his	host institution		
hnr	honouree of item		
hst	host		
ill	illustrator		
ilu	illuminator		
ins	inscriber		
inv	inventor		
isb	issuing body		
itr	instrumentalist		
ive	interviewee	interviewee (expression)	
ivr	interviewer	interviewer (expression)	
jud	judge		
jug	jurisdiction governed		
lbt	librettist		
lsa	landscape architect		
ltg	lithographer		
lyr	lyricist		
mod	moderator		
msd	musical director		
nrt	narrator	voice actor	
osp	on-screen presenter		
own	owner		
pan	panelist		
pht	photographer		
plt	platemaker		
ppt	puppeteer		
pra	praeses		
pre	presenter		
prf	performer		
prg	programmer		
prm	printmaker		
prn	production company		
pro	producer		
prs	production designer		
prt	printer		
ptf	plaintiff		
rcd	recordist		
rce	recording engineer		
rcp	addressee		
rsp	respondent		
rsr	restorationist		
scl	sculptor		
sgd	stage director		
sng	singer		
spk	speaker		
spn	sponsor		
srv	surveyor		
stl	storyteller		
tch	teacher		
ths	degree supervisor		
trc	transcriber		
trl	translator		
wac	writer of added commentary	writer of supplementary textual content	
wal	writer of added lyrics	writer of supplementary textual content	
wat	writer of added text	writer of supplementary textual content	
```

**Comment by kwahlin** [comment]
@olovy Right...and what is the connection to RDA (i.e. what does each column represent)? I've intentionally left out the strings in the "rda_app_i_1_en" column since they were simply duplicates (just lowercase) of the English prefLabels.

**Comment by olovy** [comment]
@kwahlin They're just the RDA terms for these relators.
We only use them for matching strings in imported records so I think a comment about the source is fine? Se commit above.

**Comment by kwahlin** [comment]
@olovy Sounds/looks good!

**Review by olovy — APPROVED** [review]
LGTM!

**Review by klngwll — APPROVED** [review]
🎉

File: source/languages.ttl (hunk 4 above)
Comment (poxelman) [reply source/languages.ttl]: Reply

File: source/languages.ttl (hunk 3 above)
Comment (poxelman) [reply source/languages.ttl:2142]: Reply

File: source/languages.ttl (hunk 4 above)
Comment (poxelman) [reply source/languages.ttl]: Reply

**Pull request merged**